docs/library/spi_engine/control-interface.rst

```diff
@@ -9,18 +9,18 @@ synchronization points as well as the SPI bus transmit and receive data.
 
 The interface consists of four streams:
 
-* The CMD stream which carries the SPI Engine commands. (Master to Slave)
+* The CMD stream which carries the SPI Engine commands. (Controller to Subordinate)
```

Comment:
Actually, here we are referring to the SPI Bus and not the AXI interface; so Controller is correct.
I remeber I change to subordinate because some places switched to [ref](https://learn.microsoft.com/en-us/azure-sphere/app-development/spi?view=azure-sphere-integrated) [ref](https://ece353.engr.wisc.edu/serial-interfaces/spi/) [ref](https://www.synopsys.com/dw/ipdir.php?ds=amba_ssi) for SPI; but do you think Peripheral is better for the Serial Peripheral Interface?